docs/tutorial.md

```diff
@@ -560,8 +558,8 @@ Tekton Pipelines is known to work with:
   [Stackdriver](https://cloud.google.com/logging/).
 - See [docs on getting logs from Runs](logs.md)
 
-Elasticsearch, Beats and Kibana can be deployed locally as a means to view logs:
-an example is provided at
+We can deploy ElasticSearch, Beats, and Kibana locally for logging.  For
```

Comment:
Who is "we" in this sentence?

Reply:
Hi
If you update this sentence, please keep Elasticsearch instead of ElasticSearch.

Thanks

Reply:
Hi
If you update this sentence, please keep Elasticsearch instead of ElasticSearch.

Thanks